com.sap.cloud.lm.sl.cf.persistence/src/main/java/com/sap/cloud/lm/sl/cf/persistence/jclouds/providers/aliyun/AliOSSApi.java

```diff
@@ -0,0 +1,34 @@
+package com.sap.cloud.lm.sl.cf.persistence.jclouds.providers.aliyun;
+
+import com.aliyun.oss.OSS;
+import com.aliyun.oss.OSSClientBuilder;
+import com.google.common.base.Supplier;
```

Comment:
Is it required this guava supplier to be used?

Reply:
In fact yes! Otherwise we get something like "unsatisfied dependency failure" during DS start